types/ansicolors/index.d.ts

```diff
@@ -3,8 +3,9 @@
 // Definitions by: Benjamin Arthur Lupton <https://github.com/balupton>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-interface Colors extends String {
+interface Colors {
 	(value: string): string
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/ansicolors/index.d.ts;
  Line: 7;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.